discord/gateway.py

```diff
@@ -208,6 +209,9 @@ def ack(self):
 
 
 class VoiceKeepAliveHandler(KeepAliveHandler):
+    if TYPE_CHECKING:
+        ws: DiscordVoiceWebSocket
```

Comment:
Can't you do something like ?
```suggestion
    ws: "DiscordVoiceWebSocket"
```

Reply:
It looks weird